src/app/organization/password-preferences/password-preferences.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="passwordPreferencesForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <mat-radio-group fxLayout="column" formControlName="validationPolicyId" fxLayoutGap="10px">
+          <mat-radio-button *ngFor="let passwordPreference of passwordPreferencesData" [value]="passwordPreference.id">
+            <span class="description-wrap">{{ passwordPreference.description }}</span>
```

Comment:
This is required as by default, the labels of mat-radio-button will overflow its parent content rather than wrap around it.